modules/backup-and-restore/pages/point-in-time-restore.adoc

```diff
@@ -0,0 +1,76 @@
+= Point-in-Time Restore
+
+The introduction of Point-in-Time Restore in version 4.1.0 is designed to enable users to restore the database to a previous time point, they can provide a timestamp and restore to it using previous made backups, even though no backup has been conducted at exactly that time point.
+This feature is based on xref:tigergraph-server:backup-and-restore:differential-backups.adoc[Differential Backup].
+
+== Usage
```

Comment:
@liangdeLitg  Include a "Prerequisites" section to specify the necessary conditions if there are any. What about required privileges?

Reply:
I think it has no difference to what backup/restore needs. @runlilong Do you have more info?

Reply:
@liangdeLitg  yes, there is no difference to what /backup/restore needs
cc @arun-TG-PM